run.py

```diff
@@ -1021,36 +1021,50 @@ async def github(request):
+
+        # We only accept this from people which are member of the org
+        # https://docs.github.com/en/rest/reference/orgs#check-organization-membership-for-a-user
+        # We need a token an we can't rely on "author_association" because sometimes, users are members in Private,
+        # which is not represented in the original webhook
+        async def is_user_in_organization(user):
+            token = open("./github_bot_token").read().strip()
+            async with aiohttp.ClientSession(headers={"Authorization": f"token {token}", "Accept": "application/vnd.github.v3+json"}) as session:
+                resp = await session.get(f"https://api.github.com/orgs/YunoHost-Apps/members/{user}")
+                return resp.status == 204
+
+        if not await is_user_in_organization(hook_infos["comment"]["user"]["login"]):
+            # Unauthorized
+            abort(403, "Unauthorized")
+        type = "issue"
+
+    elif hook_type == "pull_request":
+        if hook_infos["action"] != "opened":
         # Nothing to do but success anyway (204 = No content)
-        abort(204, "Nothing to do")
-
-    # Check the comment contains proper keyword trigger
-    body = hook_infos["comment"]["body"].strip()[:100].lower()
-    triggers = ["!testme", "!gogogadgetoci", "By the power of systemd, I invoke The Great App CI to test this Pull Request!"]
-    if not any(trigger.lower() in body for trigger in triggers):
-        # Nothing to do but success anyway (204 = No content)
-        abort(204, "Nothing to do")
-
-    # We only accept this from people which are member of the org
-    # https://docs.github.com/en/rest/reference/orgs#check-organization-membership-for-a-user
-    # We need a token an we can't rely on "author_association" because sometimes, users are members in Private,
-    # which is not represented in the original webhook
-    async def is_user_in_organization(user):
-        token = open("./github_bot_token").read().strip()
-        async with aiohttp.ClientSession(headers={"Authorization": f"token {token}", "Accept": "application/vnd.github.v3+json"}) as session:
-            resp = await session.get(f"https://api.github.com/orgs/YunoHost-Apps/members/{user}")
-            return resp.status == 204
-
-    if not await is_user_in_organization(hook_infos["comment"]["user"]["login"]):
-        # Unauthorized
-        abort(403, "Unauthorized")
+            abort(204, "Nothing to do")
+        # We only accept PRs that are created by github-action bot
+        if hook_infos["pull_request"]["user"]["login"] != "github-actions[bot]":
+            # Unauthorized
+            abort(403, "Unauthorized")
+        type = "pull_request"
+    else:
+       # Nothing to do but success anyway (204 = No content)
+       abort(204, "Nothing to do")
 
     # Fetch the PR infos (yeah they ain't in the initial infos we get @_@)
-    pr_infos_url = hook_infos["issue"]["pull_request"]["url"]
+    pr_infos_url = hook_infos[type]["url"]
```

Comment:
Eeeeh that's weird, are you sure it still works for comments on PR ? Pretty sure in the comment-from-PR case, we gotta access it with `hook_infos["issue"]["pull_request"]` ... that's a bit annoying to manage, I guess we gotta do something like : 

```suggestion
    if type == "issue":
         hook_infos = hook_infos["issue"]
    pr_infos_url = hook_infos["pull_request"]["url"]
```